frontend/src/containers/RecipePage/DeleteIcon/DeleteIcon.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './DeleteIcon.css';
+
+const DeleteIcon = () => (
+  <div id="delete-icon-container">
+    <div id="delete-icon-x-container" />
+    <div id="delete-icon-x-1" />
+    <div id="delete-icon-x-2" />
```

Comment:
What's actually going on here?

Reply:
Never mind, I figured it out. We're using icons8 for many of our icons on the site, so you might check that out. Also, most of these things we already have elsewhere on the site, like X, spoons, clocks, difficulty bars, etc. I'd check out the recipe card and the admin page for some of this stuff.